ui/src/search.ts

```diff
@@ -1,7 +1,9 @@
 import { writable } from "svelte/store";
 
+import * as api from "./api";
 import * as event from "./event";
-
+import * as remote from "./remote";
+// import * as waitingRoom from "./waitingRoom";
```

Comment:
```suggestion
```

Reply:
I wanna leave it there so we reintroduce it when we _actually_ respond with waiting room data, if that's cool with you?